### ISSUE TITLE
npm i bcash failed

### ISSUE THREAD

**Issue by paOol** [bcoin-org/bcash#22]
server os: ubuntu 16.04.4 LTS
node version 8.9.1
npm 5.6.0

I ran  `npm i -g bcash --production`
it returned
```
+ bcash@0.0.0
added 1 package in 0.804s
```
```
bcash
zsh: command not found: bcash
```

perhaps it wasn't published on npm yet?

**Comment by bucko13** [comment]
That's right, not published yet, but using the instructions to install from a git clone should work fine (use `staging` branch for now). We'll be testing thoroughly in the next week to get something we feel comfortable publishing as an official release asap.

**Comment by tuxcanfly** [comment]
Fixed, `v1.0.2` is available on `npm`.
